tests/e2e/specs/frontend/checkout.test.js

```diff
@@ -36,12 +36,7 @@ describe( `${ block.name } Block (frontend)`, () => {
 	let productPermalink;
 
 	beforeAll( async () => {
-		//prevent CartCheckoutCompatibilityNotice from appearing
-		await page.evaluate( () => {
-			localStorage.removeItem(
-				'wc-blocks_dismissed_compatibility_notices'
-			);
-		} );
+		// prevent CartCheckoutCompatibilityNotice from appearing
```

Comment:
Unrelated but is this needed here? CartCheckoutCompatibilityNotice shouldn't appear in checkout to begin with.

Reply:
goToCheckoutBlock method goes to the editor, on the Checkout block page to get the permalink in order to navigate to that url